### PR TITLE
Build Roslyn against 15.3

### PR DESCRIPTION
**Pull request by rchande**
Merges features/ins into master. The merge into features/ins was green: https://github.com/dotnet/roslyn/pull/19752

Tag @dotnet/roslyn-ide @jaredpar @jasonmalinowski

### CODE REVIEW

**Comment by rchande** [comment]
VSI tests are probably going to fail since master isn't running on the 15.3 pool yet.

File: src/EditorFeatures/Next/Structure/RoslynBlockTag.cs [hunk 1]
```diff
@@ -9,9 +9,9 @@
 using Microsoft.VisualStudio.Text.Tagging;
 using Roslyn.Utilities;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```
Comment (jaredpar) [inline src/EditorFeatures/Next/Structure/RoslynBlockTag.cs]: Do we have a bug tracking removing this `#pragma`?

File: src/EditorFeatures/TestUtilities/TestExportJoinableTaskContext.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using System;
```
Comment (jaredpar) [inline src/EditorFeatures/TestUtilities/TestExportJoinableTaskContext.cs]: Missing copyright header.

Comment (tannergooding) [inline src/EditorFeatures/TestUtilities/TestExportJoinableTaskContext.cs]: We should really have an analyzer for this...

Comment (jaredpar) [inline src/EditorFeatures/TestUtilities/TestExportJoinableTaskContext.cs]: We do.  It's called JaredCop.

**Comment by tmat** [comment]
Nice!

**Pull request merged**